packages/babel/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@webpack-blocks/babel",
+  "name": "@boxfoot/webpack-blocks-babel",
```

Comment:
I spy with my little eye... 😉

Reply:
Oops!  Fixed.  Sorry for the slow turnaround!